l10n_pa_withholding/model/account_invoice.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+from openerp import api, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.model
+    def wh_tax_account(self, line):
+        """Retrieves the account to use as Withholding Account"""
+        return line.account_id.id
+
+    @api.model
+    def wh_move_line_get_item(self, line):
+        sign = 1
+        if line.invoice_id.type in ('out_invoice', 'in_invoice'):
```

Comment:
this methodology is only valid for out_invoice and out_refund not for in_invoice that case has not been discussed yet.